recipes/libmysqlclient/all/conandata.yml

```diff
@@ -31,10 +31,3 @@ patches:
     - patch_file: "patches/0006-fix-cpp20-build-8.0.29.patch"
       patch_description: "Fix C++20 compilation"
       patch_type: "portability"
-  "8.0.25":
```

Comment:
I guess you also want to remove 8.0.25 under `sources` field.

Reply:
Good catch